test/ode/ode_ssprk_tests.jl

```diff
@@ -228,6 +228,34 @@ sol = solve(test_problem_ssp, alg, dt=8/5, adaptive=false)
 sol = solve(test_problem_ssp_inplace, alg, dt=8/5, adaptive=false)
 @test mapreduce(t->all(0 .<= sol(t) .<= 1), (u,v)->u&&v, range(0, stop=8, length=50), init=true)
 
+alg = SSPRKMSVS32()
+for prob in test_problems_only_time
+  sim = test_convergence(dts, prob, alg)
+  # higher order as pure quadrature
+  @test abs(sim.𝒪est[:final]-1-OrdinaryDiffEq.alg_order(alg)) < testTol
```

Comment:
why subtract by 1? It looks like this causes the tests to fail

Reply:
My mistake. I thought I had pushed that change. I''ll just do that